api_tabular/utils.py

```diff
@@ -3,6 +3,23 @@
 
 from api_tabular import config
 
+TYPE_POSSIBILITIES = {
+    "string": ["compare", "contains", "differs", "exact", "in", "sort"],
```

Comment:
Isn't it weird to compare with less/greater on strings?

Reply:
My intent was to setup the `greater/less` operators for all types that have and inherent order, which goes along with the `sort` operator. A basic use I could see could be for any king of id, which a user would like to retrieve pre-filtrered

Reply:
Yep, let's keep it if it can makes sense for someone out here